.github/ct-install.yaml

```diff
@@ -15,6 +15,7 @@ excluded-charts:
   - charts/stable/foundryvtt
   - charts/stable/heimdall
   - charts/stable/multus
+  - charts/stable/mqtt-exporter
```

Comment:
Could you elaborate a bit on the reason why the chart has to be excluded from testing? If it is because of an MQTT server being required, maybe that runtime dependency can be satisfied by added it as an additionalContainer under a `ci/ct-values.yaml` file.

Reply:
yes it was because an mqtt server is required but I might be able to include it in the ct values...